apps/extension/src/routes/page/onboarding/generate.tsx

```diff
@@ -60,6 +114,21 @@ export const GenerateSeedPhrase = () => {
               isSuccessCopyText
             />
           </div>
+
+          {reveal && (
+            <div className='mt-4 rounded-lg border border-gray-500 bg-gray-800 p-4 shadow-sm'>
+              <h4 className='text-lg font-semibold text-gray-200'>Important!</h4>
```

Comment:
suggestion: Think we can get rid of this headline and graduate block height line to an `h4`.

Reply:
great suggestion, looks much cleaner

<img width="496" alt="Screenshot 2024-09-01 at 11 20 12 AM" src="https://github.com/user-attachments/assets/8375c60e-a375-4114-aa00-6562d530b45d">